lib/bonny/controller.ex

```diff
@@ -43,12 +44,37 @@ defmodule Bonny.Controller do
         name = Enum.at(module_components, 0)
         version = module_components |> Enum.at(1, "v1") |> String.downcase()
 
-        %Bonny.CRD{
+        crd = %Bonny.CRD{
           group: @group || Bonny.Config.group(),
           scope: @scope || :namespaced,
           version: @version || version,
           names: @names || crd_spec_names(name)
         }
```

Comment:
```suggestion
          additionalPrinterColumns: additional_printner_columns()
        }
```

Move the `case` logic to its own function and return `nil` or the `list`

Reply:
done